### DISCUSSION TITLE
급함!]유니티 에디터 멀티플레이 서버 질문 있습니다.

### DISCUSSION THREAD

**Discussion by smartbh** [naverz/zepeto-studio-kor#36] [5. Q&A]
![image](https://user-images.githubusercontent.com/43138813/144887723-933ea282-993b-4719-a625-5fd8af6af62a.png)

저번주 토요일 낮까진 잘됐었는데

저녁부터 접속이 안되더니

C# 청크 에러 메시지 뜨면서

에디터에서 실행조차 불가능합니다

어떻게 해결해야 하나요?

멀티 환경은 이렇고

전체 오류 메시지는 이렇습니다.

![image](https://user-images.githubusercontent.com/43138813/144888986-41aa7a9f-8176-449c-b3e6-695f23610b35.png)

![image](https://user-images.githubusercontent.com/43138813/144889129-a2590429-c580-40ef-8e7a-81bcfc7e2f32.png)

Custom Environment를 만들어서 해보려 해도

![image](https://user-images.githubusercontent.com/43138813/144889627-d686beda-0104-496e-8f0b-6c55d1464534.png)

이러한 오류가 뜹니다

제가 혹시 멀티 예제를 따라하면서

index파일이나 index.d, schema 파일의 내용을

똑같이 따라하지 못한건지 싶어서

내용을 다시 한번 확인해봤는데

![image](https://user-images.githubusercontent.com/43138813/144891133-fc8b701f-6100-4d74-a316-03887a6632ba.png)

이러한 문제도 뜹니다

**Comment by x4cf** [comment]
12월 7일 오전 10시 현재, 에디터와 QR코드를 통한 테스트는 작동되는것같습니다.

**Reply by x4cf** [reply]
제페토 서버의 문제인지는 확인해봐야겠지만, 제 환경에서 작동은 됩니다. 하지만 캐릭터 옷은 불러오지 못하는 문제가 있는것같군요.

**Reply by x4cf** [reply]
10시 40분 캐릭터 로딩까지 정상작동합니다.

**Comment by NoahHahm** [comment]
스키마 문제로 보이는데요 혹시 schemas.json 내용을 첨부해주실수 있을까요?

**Reply by smartbh** [reply]
감사합니다. 답변 보고 스키마와 다른 index.ts, index.d.ts 파일들도 모두 맞춰주어 오류가 해결되었습니다 ㅠㅠ

**Reply by NoahHahm** [reply]
schemas.json 을 apply 하시면 변동된 index.d.ts 까지 자동 적용 됩니다.
참고 부탁드려요.
